custom_components/frigate/camera.py

```diff
@@ -45,10 +55,15 @@ async def async_setup_entry(
     """Camera entry setup."""
 
     frigate_config = hass.data[DOMAIN][entry.entry_id][ATTR_CONFIG]
+    frigate_client = get_client_for_frigate_instance_id(
```

Comment:
There's a much easier way in this instance, the client will just be `hass.data[DOMAIN][entry.entry_id][ATTR_CLIENT]`.